themes/default/content/docs/reference/pulumi-yaml.md

```diff
@@ -8,49 +8,121 @@ menu:
     weight: 5
 ---
 
-The `Pulumi.yaml` project file specifies metadata about your project, such as the project name, applicable runtime for your program, and other higher-level information.
+The `Pulumi.yaml` project file specifies metadata about your project, such as the project name and language runtime for your project.
 
-It contains the following required and optional attributes:
+## Example project files
```

Comment:
If we're not Using Title Case for Headings, we should indicate this in the Style Guide. (I don't have a strong preference either way - just that we are consistent.) It looks like our existing docs do use proper title case.

Reply:
oh lol, ill just align in this pr but we should revisit this

Reply:
https://github.com/pulumi/pulumi-hugo/pull/1834

Reply:
Makes this consistent with the wording in the Registry:

```suggestion
## Example Usage
```

Reply:
@jkodroff hmm, id rather this be specific to reduce confusion then align with the docs in the registry. am i missing something about why this needs to align with registry?

Reply:
also, would you be totally against me keeping the headings as is and me going to fix up the rest, or would you rather these be title case for now?

Reply:
Doesn't _need_ to align with the registry, but I thought it read better and was consistent to boot. No hard feelings if you feel differently.

Not totally against keeping the headings as-is since this PR is such a significant improvement over what came before.